src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure ServiceBus Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.1.0")]
```

Comment:
@v-Ajnava usually you increase AssemblyFileVersion, in this case increase AssemblyFileVersion and no need to increase AssemblyVersion